core/controllers/reader.py

```diff
@@ -1040,3 +1040,29 @@ def get(self):
             'question_dicts': question_dicts
         })
         self.render_json(self.values)
+
+
+class LearnerAnswerInfoSubmissionHandler(base.BaseHandler):
```

Comment:
Can we use the above `LearnerAnswerInfoHandler.put` for answer submission?

Reply:
Should I rename the controller class name as LearnerAnswerInfoHandler?

Reply:
No I mean, you have already created a  `LearnerAnswerInfoHandler` in which we have `get` and `delete`, so can we add the `put` there. (I'm not pretty sure about it. @aks681 any thoughts?)

Reply:
Actually put is done by the learner, so I think it should be better to have a different controller.

Reply:
Since answer info submission would be done in the player view, and answer info get and delete would only be done in the editor view, I think it makes sense to split it into reader.py and editor.py. @DubeySandeep is that fine?